src/odoc/html_page.ml

```diff
@@ -16,23 +16,45 @@
 
 open Odoc_model
 
+type source = File of Fpath.t | Root of Fpath.t
+
+let pp fmt = function
+  | File f -> Format.fprintf fmt "File: %a" Fpath.pp f
+  | Root f -> Format.fprintf fmt "File: %a" Fpath.pp f
+
+let to_string f = Format.asprintf "%a" pp f
+
```

Comment:
Maybe this could be put inside a submodule? The `Html_page.to_string` and `Html_page.pp` names can be misleading, as they are not function dealing with "html pages"...